foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/helper/ClassConstants.java

```diff
@@ -157,6 +163,10 @@
     //XML Classes
     public static final Class<Document> DOCUMENT = Document.class;
 
+    // JSON Classes
+    public static final Class<JsonValue> JSON_VALUE = JsonValue.class;
+    public static final Class<JsonArray> JSON_ARRAY = JsonArray.class;
+    public static final Class<JsonObject> JSON_OBJECT = JsonObject.class;
```

Comment:
this is sort of dangerous thing to do - even usage of the logger will require json api on the classpath. Is this what we really want?

Reply:
I know. It caused a lot of build issues and `pom.xml` changes. But this was also the _systematic_ way of extending the code.
I still have to keep jakarta.json references in ConversionManager and few other classes. But I can remove them from ClassConstants and use direct JsonValue/Array/Object.class references in the code.
Will also go trough the build/tests and remove jakarta.json from poms where possible.

Reply:
Bad news is, that I was able to remove jakarta.json only from OSGITestHelper. All new POMs dependencies are still required.